pkgs/desktops/deepin/dde-daemon/default.nix

```diff
@@ -87,6 +87,7 @@ buildGoPackage rec {
   '';
 
   buildPhase = ''
+    export PAM_MODULE_DIR="$out/lib/security"
     # compilation of the nm module is failing
     #make -C go/src/${goPackagePath}/network/nm_generator gen-nm-code
```

Comment:
should this stay around?

Reply:
> should this stay around?

I suppose it should stay.

PAM authorization is being discussed at https://github.com/NixOS/nixpkgs/pull/59943#pullrequestreview-230105844, which is about a module for deepin.